### PR TITLE
Update Rise25 nomination page, link to honorees

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Since nominations are long since closed and the honorees have been announced, the old nominate page was a bit confusing/misleading. This updates it to point to the winners site and remove the redundant portions.

This is in lieu of deleting the page entirely and redirecting the URL, at Rebecca's request.

## Testing
http://localhost:8000/rise25/nominate/

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/rise25/landing.html [hunk 1]
```diff
@@ -195,66 +114,32 @@ <h3 class="r25-c-category-title">Change Agents</h3>
...
-          'width': '230',
-          'height': '153',
-          'class': 'r25-blog-image'
-        }
-      ) }}
-    </div>
-
-    <div class="r25-spread-secondary">
-      <h3>Help us spread the word!</h3>
-      {% include "mozorg/rise25/includes/social-share.html" %}
```
Comment (janbrasna) [inline bedrock/mozorg/templates/mozorg/rise25/landing.html:230]: Both occurrences of this include are removed, the include itself can be deleted too now?

File: bedrock/mozorg/templates/mozorg/rise25/landing.html [hunk 2]
```diff
@@ -116,69 +58,46 @@ <h3 class="r25-c-category-title">Advocates</h3>
...
-          innovation, steering startups and projects with a commitment to ethical
-          standards, inclusiveness and enhancing human welfare through technology.
+          They’re the architects of trustworthy AI, including engineers and data
+          scientists dedicated to the principles of open source, open data and
+          open science. They focus on technical proficiency and responsible and
+          ethical construction. Their work ensures AI is secure, accessible and
+          reliable, aiming to create tools that empower and advance society.
         </p>
       </div>
-      {% if switch('rise25-nominations-open') %}
```
Comment (janbrasna) [inline bedrock/mozorg/templates/mozorg/rise25/landing.html:177]: Do you plan on removing the switch entirely from waffle config, or chances are next year will make use of it again so better to keep it there?;)

**Review by janbrasna — COMMENTED** [review]
Thanks for keeping at least some landing! Just a couple of housekeeping Qs:

File: bedrock/mozorg/templates/mozorg/rise25/landing.html (hunk 2 above)
Comment (craigcook) [reply bedrock/mozorg/templates/mozorg/rise25/landing.html:177]: We can safely remove it from the config once this change has been deployed. Actually there are quite a few redundant switches that we can clean up. I have a work in progress to do that and I'll add this one to the list.

File: bedrock/mozorg/templates/mozorg/rise25/landing.html (hunk 1 above)
Comment (craigcook) [reply bedrock/mozorg/templates/mozorg/rise25/landing.html:230]: doh, good catch!

File: bedrock/mozorg/templates/mozorg/rise25/landing.html (hunk 2 above)
Comment (janbrasna) [reply bedrock/mozorg/templates/mozorg/rise25/landing.html:177]: The switch is already [off](https://github.com/mozmeao/www-config/blob/fc790851a9a1e3e9ff5980850c5b8d0c0ffbc5f0/waffle_configs/bedrock-prod.env#L84) which is basically the same as if it wasn't there at all.

Why I'm asking is I also keep track of what needs removing https://github.com/mozilla/bedrock/issues/14092#issuecomment-2037115750 and some already removed waiting to clean up: https://github.com/mozmeao/www-config/pull/559 that I'm slowly adding to;) Feel free to build on top of this if it helps…

File: bedrock/mozorg/urls.py [hunk 3]
```diff
@@ -146,7 +146,6 @@
     ),
     path("antiharassment-tool/", views.anti_harassment_tool_view, name="mozorg.antiharassment-tool"),
     page("rise25/nominate/", "mozorg/rise25/landing.html"),
-    page("rise25/thanks/", "mozorg/rise25/thanks.html"),
```
Comment (alexgibson) [inline bedrock/mozorg/urls.py:149]: **question:** Is it worth adding a redirect for this URL to avoid a 404 from somewhere?

**Review by alexgibson — APPROVED** [review]
r+wc 🥞

File: bedrock/mozorg/urls.py (hunk 3 above)
Comment (craigcook) [reply bedrock/mozorg/urls.py:149]: Maybe a 404 is the best response in this case? It was only intended as the landing after submitting a nomination, and all those forms are deactivated. It didn't have any real content and shouldn't have been linked or indexed anywhere else.

**Pull request merged**